webapp/tests/mocha/unit/enketo/medic-xpath-extensions.spec.js

```diff
@@ -380,4 +380,39 @@ describe('medic-xpath-extensions', function () {
       });
     });
   });
+
+  describe('validateIdNumber()', () => {
```

Comment:
Let's rename this to the name of the function it's testing

```suggestion
  describe('cht:luhn()', () => {
```